tests/e2e/test_pvc_creation_scale.py

```diff
@@ -0,0 +1,164 @@
+"""
```

Comment:
There is a test case for capturing deletion time for 1500 PVC, planning to include it in this test itself. Part of this test case after creating 1500 pvc it's getting deleted in teardown, instead of teardown will write code to delete pvc and capture deletion time too.